docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3.4"
+services:
+  docker:
+    image: homework25:latest
+    container_name: docker-compose
+    build:
+      context: .
+      dockerfile: Dockerfile
+    expose:
+      - 5000
```

Comment:
seems like we are not forwarding any how ports in this compose
```
$ ss -tlnp
State            Recv-Q           Send-Q                     Local Address:Port                      Peer Address:Port           Process           
LISTEN           0                4096                       127.0.0.53%lo:53                             0.0.0.0:*                                
LISTEN           0                128                              0.0.0.0:22                             0.0.0.0:*                                
LISTEN           0                4096                           127.0.0.1:35021                          0.0.0.0:*                                
LISTEN           0                128                                 [::]:22                                [::]:*                                
```

Reply:
fixed, try